splash/server.py

```diff
@@ -29,6 +29,8 @@ def parse_opts():
         help="port to listen to (default: %default)")
     op.add_option("-s", "--slots", type="int", default=defaults.SLOTS,
         help="number of render slots (default: %default)")
+    op.add_option("--max-timeout", type="float", default=defaults.MAX_TIMEOUT,
+        help="maximum request processing time (default: %default)")
```

Comment:
It is not maximum request processing time - maximum request processing time is set in [timeout](http://splash.readthedocs.org/en/latest/api.html#arg-timeout) GET or POST argument. `max-timeout` is a maximum allowed value for timeout.